### PR TITLE
Made it more user friendly. Also, ARCH <3

### PR DESCRIPTION
**Pull request by kloaec**
Replaced the confusing Recovery section with a detailed Ian Coleman keygen.

### CODE REVIEW

File: doc/TRY.md [hunk 1]
```diff
@@ -49,21 +56,28 @@ Liana needs `bitcoind` to communicate with the Bitcoin network. Minimum supporte
...
-Download the `bitcoind` binary from [the official website of the Bitcoin Core
+1. Download the `bitcoind` binary from [the official website of the Bitcoin Core
 project](https://bitcoincore.org/bin/bitcoin-core-24.0.1/) according to your platform (in the context
-of this guide, it is most likely `bitcoin-24.0.1-x86_64-linux-gnu.tar.gz`).
+of this guide, it is most likely `bitcoin-24.0.1-x86_64-linux-gnu.tar.gz`), and associated SHA256SUMS and SHA256SUMS.asc verification files.
+```
+curl -O https://bitcoincore.org/bin/bitcoin-core-24.0.1/bitcoin-24.0.1-x86_64-linux-gnu.tar.gz -O https://bitcoincore.org/bin/bitcoin-core-24.0.1/SHA256SUMS -O https://bitcoincore.org/bin/bitcoin-core-24.0.1/SHA256SUMS.asc
+```
+
+2. Verify the hash of Bitcoin Core.
```
Comment (darosior) [inline doc/TRY.md:66]:
```suggestion
2. Verify the hash of the downloaded archive.
```

File: doc/TRY.md [hunk 2]
```diff
@@ -49,21 +56,28 @@ Liana needs `bitcoind` to communicate with the Bitcoin network. Minimum supporte
 
 ### Download
 
-Download the `bitcoind` binary from [the official website of the Bitcoin Core
+1. Download the `bitcoind` binary from [the official website of the Bitcoin Core
 project](https://bitcoincore.org/bin/bitcoin-core-24.0.1/) according to your platform (in the context
-of this guide, it is most likely `bitcoin-24.0.1-x86_64-linux-gnu.tar.gz`).
+of this guide, it is most likely `bitcoin-24.0.1-x86_64-linux-gnu.tar.gz`), and associated SHA256SUMS and SHA256SUMS.asc verification files.
+```
+curl -O https://bitcoincore.org/bin/bitcoin-core-24.0.1/bitcoin-24.0.1-x86_64-linux-gnu.tar.gz -O https://bitcoincore.org/bin/bitcoin-core-24.0.1/SHA256SUMS -O https://bitcoincore.org/bin/bitcoin-core-24.0.1/SHA256SUMS.asc
```
Comment (darosior) [inline doc/TRY.md:63]: Thanks for the drive-by correction on 24.0.1 :)

File: doc/TRY.md [hunk 3]
```diff
@@ -102,6 +116,7 @@ git checkout 6a6983e15e4d3c8c03937f8bee040de350ce722f
 make simulate
 ```
 
+This step might take a few minutes at the first launch of the Specter emulator.
```
Comment (darosior) [inline doc/TRY.md:119]: A few dozens of seconds maybe, but a few minutes?? Anyway, it doesn't hurt so much to have it.

File: doc/TRY.md [hunk 4]
```diff
@@ -116,23 +131,16 @@ TODO: download and sig verification details
...
 Since you presumably never installed Liana, this will start the installer. Create a new wallet.
 
-Choose network Signet. For the primary key use the one from your dummy signing device, the Specter
-simulator you just started. You can do that simply by clicking on the "import" button next to the
-text input in the installer. For the number of blocks before the recovery key becomes active, you
-can choose anything valid. But prefer something small to test the case where coins are soon to
+Choose network Signet. For the primary key we will use the one from the dummy signing device you just started. Do this by simply clicking on the "import" button next to the text input in the Liana installer. For the number of blocks before the recovery key becomes active, you
+can choose anything valid. Preferably something small to test the case where coins are soon to
 become accessible on the recovery branch.
+> a block corresponds to an **average** of 10 minutes.
```
Comment (darosior) [inline doc/TRY.md:137]: Let's not have that. :)

File: doc/TRY.md [hunk 5]
```diff
@@ -116,23 +131,16 @@ TODO: download and sig verification details
...
 
-For the recovery key you could use another simulator but in this guide i'll just use a random xpub:
-`tpubDDU2vzv4Rk2kU8VjDDQBWYTb7tmSd9ddV4ERmm5VesfoaxBJQm3CyNc4fjcYAzEqXn3YQ8dxpzkhQjpxT3Nqp7yQh1UMczL1MMfTSKXNv3n/<0;1>/*`.
-You can also generate one from, for instance https://iancoleman.io/bip39/. But make sure to append
-`/<0;1>/*` to it, this is to be able to derive both change and receive addresses from the same
-descriptor.
-
-TODO: have the installer input xpubs without the derivation part. https://github.com/revault/liana/issues/155
+For the recovery key you could use another simulator but in this guide i'll just use a key generated online at https://iancoleman.io/bip39/. Click the **GENERATE** at the top of the website, then make sure to select **Coin: BTC-Bitcoin Testnet** to generate a tpub_. You can then copy the **Account Extended Public Key** and paste it in the recovery path of Liana.
```
Comment (darosior) [inline doc/TRY.md:139]: Nice. Thanks.

```suggestion
For the recovery key you could use another simulator but in this guide i'll just use a key generated online at https://iancoleman.io/bip39/. Click the **GENERATE** at the top of the website, then make sure to select **Coin: BTC-Bitcoin Testnet** to generate a tpub. You can then copy the **Account Extended Public Key** and paste it in the recovery path of Liana.
```

File: doc/TRY.md [hunk 6]
```diff
@@ -116,23 +131,16 @@ TODO: download and sig verification details
...
 
-Make the next step happy by ticking the "i backed up my descriptor" checkbox.
+Make the next step happy by ticking the "I backed up my descriptor" checkbox.
 
-Import the descriptor to your Specter by clicking on the `specter-simulator` tile.
+Import the descriptor to your Specter by clicking on the `specter-simulator` tile. Accept it on the Specter emulator.
```
Comment (darosior) [inline doc/TRY.md:143]: :+1:

File: doc/TRY.md [hunk 7]
```diff
@@ -2,8 +2,10 @@
 
 > *Just give me the TL;DR!*
 
-This document is a short set of instructions for trying out Liana on signet. It does not attempt to
+This document is a short set of instructions for trying out Liana on Bitcoin signet. It does not attempt to
 give any nuance, details or describe alternative configurations.
+This guide uses an emulator of the Specter hardware signer. You can also use an actual Specter or Ledger hardware should you have one at hand.
+**Do not use the Specter emulator on the mainnet/with real funds. This guide is for a test network, Signet**
```
[the remark this answers is not in the document]
Comment (kloaec) [reply doc/TRY.md:8]: I'm quite certain some rando will be happy with his test, and re-do it on main-net with the same setup.

[the remark this answers is not in the document]
Comment (darosior) [reply doc/TRY.md:8]: I'm not. :) I don't think this is the place to tell people not to use temporary keys to receive bitcoins, not to send coins to random addresses found on the internet, not to sign transactions with SIGHASH_NONE, not to [enter one of the millions ways of burning your coins].

File: doc/TRY.md [hunk 8]
```diff
@@ -2,8 +2,10 @@
 
 > *Just give me the TL;DR!*
 
-This document is a short set of instructions for trying out Liana on signet. It does not attempt to
+This document is a short set of instructions for trying out Liana on Bitcoin signet. It does not attempt to
 give any nuance, details or describe alternative configurations.
+This guide uses an emulator of the Specter hardware signer. You can also use an actual Specter or Ledger hardware should you have one at hand.
```
[the remark this answers is not in the document]
Comment (kloaec) [reply doc/TRY.md:7]: ok

File: doc/TRY.md (hunk 3 above)
Comment (kloaec) [reply doc/TRY.md:119]: Yes, about 5 min for both Maria and I if i recall well

Comment (darosior) [reply doc/TRY.md:119]: Ok
